packages/terra-hyperlink/src/Hyperlink.module.scss

```diff
@@ -39,90 +43,31 @@
-    }
-  }
-
-  .video {
-    &::after {
-      background-image: var(--terra-hyperlink-video-after-background-image, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#0059a8" d="M48 10.9v26.3l-10-10v-6.3l10-10zm-13 .4C35 10 34 9 32.7 9H2.3C1 9 0 10 0 11.3v25.4C0 38 1 39 2.3 39h30.4c1.3 0 2.3-1 2.3-2.3V11.3z"></path></svg>'));
-    }
-  }
-
-  .video.is-disabled {
-    &::after {
-      background-image: var(--terra-hyperlink-video-disabled-after-background-image, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="rgba(28, 31, 33, 0.2)" d="M48 10.9v26.3l-10-10v-6.3l10-10zm-13 .4C35 10 34 9 32.7 9H2.3C1 9 0 10 0 11.3v25.4C0 38 1 39 2.3 39h30.4c1.3 0 2.3-1 2.3-2.3V11.3z"></path></svg>'));
-    }
-  }
-
-  .audio {
-    &::after {
-      background-image: var(--terra-hyperlink-audio-after-background-image, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#0059a8" d="M29 0v48L15 34H0V14h15L29 0zm6.3 12.7l-2.1 2.1c2.3 2.3 3.8 5.6 3.8 9.2 0 3.6-1.5 6.8-3.8 9.2l2.1 2.1C38.2 32.4 40 28.4 40 24s-1.8-8.4-4.7-11.3zM41 7l-2.1 2.1C42.6 13 45 18.2 45 24c0 5.8-2.4 11-6.2 14.8L41 41c4.3-4.3 7-10.3 7-17s-2.7-12.6-7-17z"></path></svg>'));
+    &:hover {
+      color: var(--terra-hyperlink-underline-hidden-hover-color, #001f67);
+      text-decoration: underline;
     }
-  }
 
-  .audio.is-disabled {
-    &::after {
-      background-image: var(--terra-hyperlink-audio-disabled-after-background-image, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="rgba(28, 31, 33, 0.2)" d="M29 0v48L15 34H0V14h15L29 0zm6.3 12.7l-2.1 2.1c2.3 2.3 3.8 5.6 3.8 9.2 0 3.6-1.5 6.8-3.8 9.2l2.1 2.1C38.2 32.4 40 28.4 40 24s-1.8-8.4-4.7-11.3zM41 7l-2.1 2.1C42.6 13 45 18.2 45 24c0 5.8-2.4 11-6.2 14.8L41 41c4.3-4.3 7-10.3 7-17s-2.7-12.6-7-17z"></path></svg>'));
-    }
-  }
+    &:visited,
+    &.is-visited {
+      color: var(--terra-hyperlink-underline-hidden-visited-color, #006fc3);
+      text-decoration: underline;
 
-  .document {
-    &::after {
-      background-image: var(--terra-hyperlink-document-after-background-image, inline-svg('<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><path fill="#0059a8" d="M41.9 9.9l-8.8-8.8A3.12 3.12 0 0 0 30.6 0H14.2A2.56 2.56 0 0 0 12 2v5H7a2.57 2.57 0 0 0-2 2v37a2.24 2.24 0 0 0 2 2h26a2.07 2.07 0 0 0 2-2v-5h6.1a2.06 2.06 0 0 0 1.9-2V12a2.92 2.92 0 0 0-1.1-2.1zM31 3.2l9 8.8h-9zM32 45H8V10h4v29a2.25 2.25 0 0 0 2.1 2H32zm8-7H15V3h13v9a2.77 2.77 0 0 0 3.1 3H40z"></path></svg>'));
+      &:focus {
+        color: var(--terra-hyperlink-underline-hidden-visited-focus-color, #001f67);
+      }
     }
   }
 
-  .document.is-disabled {
```

Comment:
Are the disabled states appropriately handled with introducing terra-icons and inheriting certain dithering and opacity? Previously the disabled states had different icons.

Reply:
I'd think that the current change is expected, going by @neilpfeiffer's comment on the issue.

> This will allow the icons to be able to take the current-color of the text for all states (hover, active, visited, inactive, etc.)

Neil, I am sorry if I have misquoted but could you please confirm it?

Reply:
That is what he's saying. And that's working.